web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.js

```diff
@@ -1,5 +1,42 @@
+import { Box, Stack, Typography } from "@mui/material";
+import whatIsWcsStyles from "./what_is_wcs.styles";
+import messages from "./messages";
+
 function WhatIsWcs() {
-  return <div>What is west coast swing</div>;
+  return (
+    <Box sx={whatIsWcsStyles.whatIsWcsStylesContainer}>
+      <Typography variant="h4" sx={whatIsWcsStyles.header}>
+        What is West Coast Swing?
+      </Typography>
+      <Box sx={whatIsWcsStyles.itemsContainer}>
+        <Stack sx={whatIsWcsStyles.twoColumnContent}>
+          <Box sx={whatIsWcsStyles.videoContainer}>
+            <iframe
```

Comment:
Might be nice to extract this into it's own component

Reply:
done